### ISSUE TITLE
[Idea] War Days

### ISSUE THREAD

**Issue by watergod1234** [EarthMC/Issue-Tracker#444]
Hello! There have been numerous posts lately about having PVP enabled in towns perminately or for a couple weeks as a trail. As a compromise to players who don’t want PVP enabled all the time I propose maybe only on Saturdays (could be any day, it doesn’t matter) PVP would be enabled in all towns. It’d be like a dedicated war day (or the next closest thing) for nations to siege and kill their enemies on a periodic basis. Since everyone would know when it is people would prepare or choose not to go outside during wartime. To me, this seems like a good compromise to everything. Not having any war type function makes the geopolitics on a “geopolitical sandbox” kind of stale in my opinion.

**Comment by BowShot118** [comment]
War days used to be a thing on classic, I don't know why they were disabled.
(Well Probably lag)
It would be nice to see them come back so long as you have a way to opt out xD

**Comment by watergod1234** [comment]
You can opt out by not attacking anyone or just hiding inside. Players won’t be able to break blocks still.

**Comment by Ultil0l** [comment]
War days would be epic

**Comment by watergod1234** [comment]
Yeah and the best thing is that it is already available within the restrictions of the towny plugin. This means they wouldnt have to do a thing development-wise.

**Comment by Kyoris** [comment]
i support this idea

**Comment by alekb1** [comment]
I support this idea!

**Comment by Ultil0l** [comment]
I support this idea!!

**Comment by watergod1234** [comment]
I support this idea!!!

**Comment by Pengun78** [comment]
Cant we have the towny war plugin, where you can actually take lands instead of having to fight in claims without purpose

**Comment by BowShot118** [comment]
Flag wars are shit. I saw them being used on WorldMC (Tailz's gay server).
While being on the winning side, it was fun, (now idc about Conman), but the loosing side was fucking obliterated and they lost everything. It's a broken plugin that allows for a toxic mess.

**Comment by Pengun78** [comment]
The number 1 thing we all want is a war plugin except we get lag😎😎😎😎😎

**Comment by 4noah18** [comment]
despite your mom being gay, water, this is a good idea and i **agree**

**Comment by watergod1234** [comment]
> Cant we have the towny war plugin, where you can actually take lands instead of having to fight in claims without purpose

The purpose is for the pure fun of fighting. Games shouldn't be all about incentive all the time. Some times I just go out and kill people just for fun. But I agree, it would be nice if there were a separate plugin but this is already possible within the limitations of the towny plugin and could be done with little to no effort development-wise. 



> despite your mom being gay, water, this is a good idea and i **agree**

Thanks I'm glad you agree!

**Comment by JuliusCaesar12** [comment]
> The purpose is for the pure fun of fighting. Games shouldn't be all about incentive all the time. Some times I just go out and kill people just for fun. But I agree, it would be nice if there were a separate plugin but this is already possible within the limitations of the towny plugin and could be done with little to no effort development-wise.

Firstly, for killing people just for fun, you should be stabbed 27 times. Secondly, I would like to see a world war in EMC just for once so we can claim new lands.

**Comment by Kyoris** [comment]
Talking about a world war... How about we save a backup of the server and then start a world domination game to see who stands out on top after say two weeks... Then reload the save and continue on as if nothing has happened...

**Comment by watergod1234** [comment]
Yeah dude. I enjoy fighting people. There’s no reason that should be looked down upon. It’s fun and thrilling to fight against a similarly skilled and geared opponent. The only thing that ruins these encounters is running into claims, which could be solved by the proposition I have proposed.

**Comment by Pengun78** [comment]
Its just that fighting wont give lands, neither damage, just pvping and sometimes 1 of them dies, the other side can just randomly stop pvping and nothing changes

**Comment by Pengun78** [comment]
We need an actual war plugin but itll never me made which makes emc more and more boring every day for me

**Comment by watergod1234** [comment]
This is a temporary solution that can be brought RIGHT NOW. Would you rather have this or nothing?

**Comment by Pengun78** [comment]
Ur right, but i dount itll be temporary, i actually think forever as no one that can code is taking the time to make a war plugin

**Comment by watergod1234** [comment]
> Ur right, but i doubt itll be temporary, i actually think forever as no one that can code is taking the time to make a war plugin

Even so this is the best alternative.

**Comment by Pengun78** [comment]
I agree

**Comment by HaydenTownley** [comment]
I dont belive this is a good soultion in the slightest. As much as there is a need a soultion for dealing with conflicts but a dedicated day where you arnt allowed to be safe even in your town is not the way. That is asking for players to go around harassing and just attacking people for the lols adn kinda ruins one full day of the server for people. At best if they add this it wouldnt be a weekend day and it would be Monday-Thursday which would impact players less but its not what the server is about.  Im all for a soultion to resolving conflicts but this isnt it.

**Comment by Pengun78** [comment]
I do live the idea i just think towns would team up and go hunt and like 4v1 people

**Comment by BowShot118** [comment]
> I dont belive this is a good soultion in the slightest. As much as there is a need a soultion for dealing with conflicts but a dedicated day where you arnt allowed to be safe even in your town is not the way. That is asking for players to go around harassing and just attacking people for the lols adn kinda ruins one full day of the server for people. At best if they add this it wouldnt be a weekend day and it would be Monday-Thursday which would impact players less but its not what the server is about. Im all for a soultion to resolving conflicts but this isnt it.

It's a "war day," a server event. This may be once per week and what not. But, I do very much like Kyoris' idea

**Comment by HaydenTownley** [comment]
Pock im refering to once a week thing. If you want a war day like on classic then thats fine but the server wont be able to handle the lag. But once a week war is no fun. It just makes it so people avoid the server for a day if they dont want to be attacked.

**Comment by Pengun78** [comment]
Without any sort of event like this, people can pvp what they want but it wont change anything. For example nation1 takes land of nation2 by making a town in their territory, nation2 attacks but that doesnt mean the town will go. Nation1 could also hide in claims

**Comment by BowShot118** [comment]
StOp CaLlInG mE PoCk.

idk who u r

**Comment by Pengun78** [comment]
Ill always call u pock 😎😎😎😎

**Comment by BowShot118** [comment]
> Ill always call u pock 😎😎😎😎

You're epic 😎

**Comment by Ultil0l** [comment]
why is this chat full of retard pacifists
Just assume you are shit at pvp just like me <o/

**Comment by watergod1234** [comment]
> I dont belive this is a good soultion in the slightest. As much as there is a need a soultion for dealing with conflicts but a dedicated day where you arnt allowed to be safe even in your town is not the way. That is asking for players to go around harassing and just attacking people for the lols adn kinda ruins one full day of the server for people. At best if they add this it wouldnt be a weekend day and it would be Monday-Thursday which would impact players less but its not what the server is about

It's the only solution that doesn't require any effort and could be done **right now.** If don't want to get attacked just don't make any enemies or stay inside. In my eyes a majority of the community wants some sort of PVP war-like function like this instead of gearing up for nothing.

**Comment by Aitonater** [comment]
I've been on a towny server had pvp enabled on the weekends. It worked well, towns that had good walls allowed residents to continue as normal and even then you could still hide inside your house.

**Comment by HaydenTownley** [comment]
I dont belive most of the server wants a pvp war like function. There is a good part of the server but no way a majority. The staff is failing to help with people with that playstyle but we cant have a system that stops others from playing there normal playstyle and force you to play a specfic way.

**Comment by watergod1234** [comment]
> I dont belive most of the server wants a pvp war like function. There is a good part of the server but no way a majority. The staff is failing to help with people with that playstyle but we cant have a system that stops others from playing there normal playstyle and force you to play a specfic way.
 
Every single person with the exception of you seems to favor this idea or wants a similar war system. Tell me, how is that not a majority?

**Comment by Aitonater** [comment]
It's not a majority because its only people who have viewed this thread which is a very very small sample pool, make a straw pole on earthmc reddit and share it everywhere.

**Comment by watergod1234** [comment]
Well maybe but even so, this is a good compromise to satisfy both sides. Only 14% of the time PVP would be enabled in towns. This isn’t perfect but the best compromise.

**Comment by watergod1234** [comment]
Also the players that commented on this thread for the most part are extremely active and experienced players. In the community of these players, war is a highly wanted feature. This proposition also requieres little to no stakes so if you lost the worst that would happen is that you die. Nothing happens to your town, nothing is destroyed, no fuss.

**Comment by BowShot118** [comment]
> > I dont belive most of the server wants a pvp war like function. There is a good part of the server but no way a majority. The staff is failing to help with people with that playstyle but we cant have a system that stops others from playing there normal playstyle and force you to play a specfic way.
> 
> **Every single person** who has commented on this thread with the exception of you seems to favor this idea or wants a similar war system. Tell me, how is that not a majority?

I'm against war, always have been. I just see this as an easy and fair solution to people who want war, and to people who don't. War events are fun, and they're actually okay as they aren't a 24/7 occurrence.  

> It's not a majority because its only people who have viewed this thread which is a very very small sample pool, make a straw pole on earthmc reddit and share it everywhere.

He makes a point ^^



> Also the players that commented on this thread for the most part are extremely active and experienced players. In the community of these players, war is a highly wanted feature. This proposition also requieres little to no stakes so if you lost the worst that would happen is that you die. Nothing happens to your town, nothing is destroyed, no fuss.

The Reddit... heh

**Comment by Ultil0l** [comment]
People who dont want war are usually less active <o/

**Comment by watergod1234** [comment]
> People who dont want war are usually less active <o/

Exactly. So why not satisfy the people who are playing your server the most? This is it! **cough Fix and Karl***

**Comment by JuliusCaesar12** [comment]
What they need to do is make a plugin that activates itself on a specified day and won't let people attack people from their nation or allies. Meaning you can only attack your enemies.

**Comment by Aitonater** [comment]
> What they need to do is make a plugin that activates itself on a specified day and won't let people attack people from their nation or allies. Meaning you can only attack your enemies.

Karl has said several -several- times there is no such plugin that exists or works in this fashion and that he won't be devoting time to make a war plugin.

**Comment by falconov** [comment]
I personally don't mind a war day, but I don't want to have to not play once a week because of people killing me and taking my stuff. (I am primarily a builder) If war days are to be used could we toggle keep inventory for pvp kills?

**Comment by watergod1234** [comment]
> I personally don't mind a war day, but I don't want to have to not play once a week because of people killing me and taking my stuff. (I am primarily a builder) If war days are to be used could we toggle keep inventory for pvp kills?

If you’re a builder then build a wall and you’ll be fine.

**Comment by Ultil0l** [comment]
ya

**Comment by falconov** [comment]
That would ruin my towns layout. It is a town. Not a castle

**Comment by falconov** [comment]
I am not building a wall just for someone to then go and complain that people are hiding behind walls

**Comment by Aitonater** [comment]
Then stay inside its only one day a week or don't login. Unless you have something constructive to say besides I don't like this its too much work or does not fit my town, like your town is the only one that exists, then fuck off.

**Comment by Ultil0l** [comment]
Its your fault that your town isnt walled <o/

**Comment by falconov** [comment]
I am only asking for keep inv on pvp kills during war days. I am also providing a different view point on this. I am not against it. I am simply asking for a compromise

**Comment by Aitonater** [comment]
A lot of the focus of pvp is to get items or gear for pvp. Also that would require a plugin and development so I very much doubt it would be achievable. You would have issues with people dying from fall damage or from mobs? Do they keep inventories and if not how do you make that a plugin and who would develop it?

**Comment by alekb1** [comment]
That would defeat the intire purpose. The whole point is to hurt the enemy nation, and the best way to do that is by taking away their god gear.

**Comment by falconov** [comment]
They already have keep inv on for falling into the void, so pvp kills should be easy.
Also, I don't want to be building and then some trigger happy twat kills me and takes my mats.

**Comment by Aitonater** [comment]
If its so easy make a plugin because I don't see the server devs taking the time to do it when they're trying to upgrade and fix basic bugs like shop signs and concrete.

**Comment by alekb1** [comment]
You really have three good solutions, either build a fence, work inside, or just don’t play that one day... We’re simply asking for some meaningful war on this server, and your “compromise” would make it worthless

**Comment by falconov** [comment]
Rn claims are safe places for those who don't want to fight. If most players want to fight, why not just fight outside towns?
The way I see it most people want to keep claims as safe havens. That is the current rule and if you don't like it, you can sod right off

**Comment by Aitonater** [comment]
The compromise is not having war 24/7 and having it on one day a week.

**Comment by Aitonater** [comment]
Falcon fuck off you are contributing nothing to this discussion that is meaningful or helpful. People cheat is the answer when they pvp and claims have protection thats why the war day is being put forward.

**Comment by Ultil0l** [comment]
just dont get on on war days <o/

**Comment by bapsachiko** [comment]
Support, 90% of our nation in China quit out of achieving practically everything there is to do, theres just no end game right now and this would be a good start to something keeping the server intresting.
Currently alliances don't mean much as there are no threats to actually group up, the political aspect of this server doesn't really exist right now other than cussing in global or claim blocking.

**Comment by Aitonater** [comment]
Yeah in light of making alliances if you are members of the same party you dont deal damage to one another, this is something you can already do in the server for anyone who didn't know.

**Comment by watergod1234** [comment]
> Support, 90% of our nation in China quit out of achieving practically everything there is to do, theres just no end game right now and this would be a good start to something keeping the server intresting.
Currently alliances don't mean much as there are no threats to actually group up, the political aspect of this server doesn't really exist right now other than cussing in global or claim blocking.
 
Exactly the reason why this is sorely needed. This is not the case only with your nation but other nations like Panama quit because of no war-like function, and there are many more individual players who have quit for this reason too.

**Comment by Aitonater** [comment]
Its worth understanding with walls that they don't need to be big they could even be a fence as long as they are far enough from your border even someone with max feather falling and a 40% speed potion won't be able to make a 10 block gap jump using height and running at max sprint  A. get over your fence and B. survive the fall damage. so if you have a fence in your border chunk or even a 5 block high wall it will not matter how fast someone runs or how high the build a tower to jump over they physically wont be able to and survive the fall damage. Sorry I didn't word this well.

**Comment by Aitonater** [comment]
Ok so I have done some testing. As long as your town wall is obv. higher than one block so people can't just step over it. And it is built on the 14th block from the edge of your claims. Then even if -if someone has speed II and feather falling IV they will die from the fall damage trying to jump far enough over your wall and -and they won't even make it to your wall. 
TLDR; make your town walls 13 blocks from the edge of claims and make them as small as you want and no one will ever be able to get over them even if they are just a wooden fence. People can jump over this stance and clear your wall but they will die everytime.

**Comment by Aitonater** [comment]
14th block inside your claim**

**Comment by watergod1234** [comment]
Yes, this goes to prove that walls are very effective.

**Comment by watergod1234** [comment]
@KarlOfDuty please comment your thoughts on this. Its received a lot of positive attention and would be a great temporary fix for the war situation.

**Comment by Ultil0l** [comment]
dab

**Comment by bapsachiko** [comment]
https://strawpoll.com/ehwp69ss

Just gonna leave this here

**Comment by CrookedToe** [comment]
I would just like to point out that you can use ender pearls to easily get over any wall or defensive structure.

**Comment by Kyoris** [comment]
Tru :thonk:

**Comment by Aitonater** [comment]
Yeah wow I am retarded I forgot enderpearls exist, however they're not super common you need a cleric to get em

**Comment by CrookedToe** [comment]
most nations have villagers that you can trade with like Romania, and most shops have them for relatively cheaply.

**Comment by alekb1** [comment]
Guys, you can’t use ender pearls in other towns... it’s disabled so people can’t get into places they were not intended to be. Which is why walls are still a valid solution since there are no elytra

**Comment by CrookedToe** [comment]
can you throw them from the outside in?

**Comment by alekb1** [comment]
Nope

**Comment by CrookedToe** [comment]
well ima test it out when i get home

**Comment by alekb1** [comment]
Sure, just remember that it cannot be your own town ;)

**Comment by CrookedToe** [comment]
Didn't have time to test it yesterday, but I have confirmed another way to make walls obsolete but I'm not going to say how cause I want to use it if this gets implemented.

**Comment by bapsachiko** [comment]
Walls or not its one or two days a week. I played on tons of servers before with pvp on everywhere at all time and there was never really a issue as you could still just stay inside houses until they left, no one is going to go out to make your life miserable every single day unless you pissed them off or are at war, both can be avoided by not joining people who involve themselves in politics & just be peaceful in general.
Yes there are people who raid constantly, but you have to remember those people attack anyone and there are 150 other people sharing the server with you most time of the day, they come and quickly leave if you get away then look for another target.
tl;dr dont involve yourself with incompetent nations dragging you into shit and dont be a dick if you dont want people to target you, its that ez

**Comment by CrookedToe** [comment]
The thing is that this is a GEOPOLITICAL simulator, you are supposed to be involved and it's next to impossible to just go and do something else, not to mention you seem to seriously underestimate how petty people are and how towns are supposed to be a safe haven for diplomacy and building, and people who want to pvp can just go outside the claim. Not to mention that walls are useless with a method I came up with, and secondly people don't want to be harassed. If people are really wanting to pvp why don't 2 sides just agree to. There is no point in forcing nations that just want to deal with diplomacy to just be attacked

**Comment by bapsachiko** [comment]
Yeah you're right, its a geopolitical simulator without any wars and threats lmao.
I know how petty people can be, but as I mentioned once it's mostly up to you which path you take.
If you want to be involved as you say, that's your choise and if you get mixed into some shit then you either win or take the L.
I've played both ways for many years now on plenty of different servers, and if I wanted to stay irrelevant without trouble the times I got raided was by far less then when I involved myself with the server.
Also with your last part, there is no diplomacy whatsoever thats actually nessecary, we pissed off the entire server and what did we suffer? Nothing, no one could retaliate.
What about recently when our relations broke down with Byzantium and a war started? Nothing either, they can't launch a attack and neither can we, the peak of our "war" was making a mudhut with signs in their capital where they forgot to claim and shitposting for half a hour before leaving.
We never ever had to engage in diplomacy, the only time we did it was for teleports and that took circa one minute to accomplish.
Its just plastic diplomacy for the sake of RP right now.

**Comment by CrookedToe** [comment]
If you actually attempt to be diplomatic it can do things, for example in Ireland we can convince towns to join us against Britain, and we formed a union with Scotland. Right now its Diplomacy with a hint of pvp, while with war days and such it would just be people in god armor camping towns all day to kill whoever. If there is a war each nation could just enable pvp in their capital which would actually be part of the rp. War days aren't going to fix the problem of no wars, they are literally useless in the scheme of geopolitics.

**Comment by bapsachiko** [comment]
If a town refuses to join you, youll never achive uniting anything.
Normally this would lead to war which is where pvp comes in and you can pressure those people.
You uniting against britain serves no role at all, it wont affect anything as everyone will continue their day with no risk and no wars, sitting still grinding the same useless gear, useless cause you can't really put it to proper use.
Neither should you get to pick if you want war or not, it comes and you deal with it, if this server wasnt lucky with youtube it would have had by far fewer people than it had in december.
The server lost a huge part of their older players and it got replaced with new ones, those same new players will eventually reach the same boring end game that the previous players did and also quit.

**Comment by Aitonater** [comment]
I don't think asking people who don't want to pvp to stay indoors or stay off the server for 24 hours is really that much of a hardship.

**Comment by Ultil0l** [comment]
just putting this back on the top to remind people this exists

**Comment by Aitonater** [comment]
If anything this might be a way of reducing server stress and allowing new players to login with a shorter queue on a saturday. A counter argument to this may be that it would result in new players being killed a lot but I think the noob protection would help reduce that.

**Comment by Ultil0l** [comment]
well new players being killed is a good thing tbh

**Comment by Aitonater** [comment]
True... but that's not a popular opinion for town growth. Increasing the town cost has not made a dent in the number of new towns or in increasing the size of already existing ones.

**Comment by Ultil0l** [comment]
Just make a town razing feature then, but that would be a whole other tread

**Comment by Aitonater** [comment]
Yeah a thread which would quickly be awnsered by karl going no

**Comment by Ultil0l** [comment]
ik

**Comment by KarlOfDuty** [comment]
I can spoil that the reason this hasn't been closed is because it is one of the things we are looking into for the server for this month. We had a big planning session over the last week of march and we decided on 4 currently unannounced projects, where one was the server move which failed spectacularly and another is looking at reintroducing war days.

I'm not saying it will be done but I am saying we are looking into it.

**Comment by Ultil0l** [comment]
That would be epic Karl, emc really needs this kind of shit

**Comment by watergod1234** [comment]
Yes! That’s great Karl!

**Comment by watergod1234** [comment]
But, you know if you’ve got war days you’ve also gotta get an anti-combat logging plugin as well. If war days do come would that be a guaranteed feature?

**Comment by Ultil0l** [comment]
Combat logging issue when?

**Comment by watergod1234** [comment]
"We used to have explicit war systems but the only two available are both buggy and have glaring issues of exploiting the system. We used these systems on and off for the first two years but as most of the community hated it after they had it used against them we decided to not use them again. Although we are planning two as of yet unannounced updates that will hopefully fill this role in the future." -karlofduty

Source: https://angry-mob.com/earthmc-minecraft-interview/ 

War is confirmed. Who’s excited?

**Comment by Aitonater** [comment]
Yeah I think this confirms it, a war system where there is an attrition of land isnt implemented to A. Protect builds and B. Because its buggy

**Comment by FranchuFranchu** [comment]
The best thing about this is that on Saturdays the queue is full so it's not worth joining anyway

**Comment by Magenent** [comment]
> They already have keep inv on for falling into the void, so pvp kills should be easy.
> Also, I don't want to be building and then some trigger happy twat kills me and takes my mats.

then don't make people mad at you if there is no reason to be attacked the odds of you being attacked are low as fuck

**Comment by Aitonater** [comment]
This will also allow a lot of new players to play on the server as they will have the noob protection they can get on and take a look around if regulars dont want to be killed

**Comment by watergod1234** [comment]
With lag being basically fixed at this point, this is the next thing the server needs. Or maybe even a higher stakes war plugin (town war works and is fun: https://www.spigotmc.org/resources/⚔-wars-for-towny-⚔-conquest-glory.65216/ ).  We were right about combat log being a liked feature by a large majority of the player base even though I recall staff saying players would dislike it if it were added. This will be the same! Just give it a try. @KarlOfDuty poke!

**Comment by Goosius1** [comment]
F.Y.I

I am currently working to add a new war system to the Towny plugin. It is called Siege-War and it is focused on supporting geo-political conflict.

Here is a link to the PR so that you can read about its features and monitor its progress: TownyAdvanced/Towny#3541

**Comment by Snowyy4K** [comment]
I support this idea!!
